src/components/Modal/components/Dialog/Dialog.tsx

```diff
@@ -54,7 +54,7 @@ export default function Dialog({
       {...props}
       mountOnEnter
       unmountOnExit
-      timeout={Duration.Base}
+      timeout={parseInt(durationBase, 10)}
```

Comment:
I thought this was already going to be an integer. Is it not?

If not - we could adapt polaris-tokens to export an int as expected.

Reply:
Looks like that is not the case 😭 

```
  durationBase: "200ms";
  durationFast: "100ms";
  durationNone: 0;
  durationSlow: "300ms";
  durationSlower: "400ms";
  durationSlowest: "500ms";
```

<img width="576" alt="Screen Shot 2019-03-29 at 10 18 14 AM" src="https://user-images.githubusercontent.com/643944/55238935-0ea09180-520c-11e9-9981-3abd4589f907.png">

Reply:
Thanks! This is a bug, let me look into it!

Reply:
Should we be writing tests to check for expected values?

Reply:
There is a set of snapshot tests, which have been pretty helpful so far, but more tests wouldn't hurt!

Reply:
Updated to `beta2` and no longer using `parseInt`